eo-runtime/src/test/java/EOorg/EOeolang/EOcageTest.java

```diff
@@ -255,6 +256,17 @@ void writesBoundedCopyOfTheSameBase() {
         );
     }
 
+    @Test
+    void throwsExceptionIfRecursion() {
+        final Phi cage = new EOcage(Phi.Φ);
+        writeTo(cage, cage);
+        Assertions.assertThrows(
+            ExAbstract.class,
+            new Dataized(cage)::take,
+            "Throws exception if recursion happened"
```

Comment:
@levBagryansky This message will be shown if test fails. So, maybe it's better to write something like instead:
```
We except the %s exception to be thrown, but nothing happened. 
```
What do you think?

Reply:
@levBagryansky Then maybe we need to add more context info to this message? Imagine a newcomer will look at your test. You can give them a clue how to understand your test.

We recently [discussed](https://github.com/volodya-lombrozo/jtcop/issues/326), that default JUnit messages might not be informative enough. Usually they are rather general.

Reply:
@volodya-lombrozo What would you like to add? As for me the test is already clear, commentaries are redundant.

Reply:
@levBagryansky 
```
We expect the %s exception to be thrown since we have recursion here, but nothing happened. 
```

Reply:
But if you believe it's redundant, I'm not insist. You might leave it as it is now.